src/modules/design/property-panel/indextabs/general/repository/general.repository.ts

```diff
@@ -45,18 +45,13 @@ export class GeneralRepository {
     return startEvents;
   }
 
-  private importXmlIntoModeler(modeler: IBpmnModeler, xml: string): Promise<void> {
-    return new Promise((resolve: Function, reject: Function): void => {
-      modeler.importXML(xml, (error: Error) => {
-        const errorOccured: boolean = error !== undefined;
-        if (errorOccured) {
-          reject();
-
-          return;
-        }
-
-        resolve();
-      });
-    });
+  private async importXmlIntoModeler(modeler: IBpmnModeler, xml: string): Promise<void> {
+    try {
+      await modeler.importXML(xml);
+
+      return Promise.resolve();
+    } catch (error) {
+      return Promise.reject(error);
+    }
```

Comment:
```suggestion
    await modeler.importXML(xml);
```

Diese Funktion wird eigentlich ziemlich unnötig und sollte man entfernen, da hier eigentlich nichts passiert, was nicht auch direkt über den entsprechenden Modeler machen könnte.